index.bs

```diff
@@ -2202,6 +2202,45 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator.
 </div>
 
+#### Registration API Exceptions
+
+[=[WRPS]=] can encounter the following {{DOMException|DOMExceptions}} from a call to {{CredentialsContainer/create()|navigator.credentials.create()}}. Some errors can have multiple reasons for why they happened, requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn:
+
+<dl dfn-type="argument" dfn-for="Registration API Exceptions">
+    :   <dfn>AbortError</dfn>
+    ::  The ceremony was cancelled via an {{AbortController}}
+        (see [[#sctn-abortoperation]] and [[#sctn-sample-aborting]] for more information.)
```

Comment:
I'm not sure what writing style guides say about whether the period should be inside or outside the parentheses (I personally prefer outside), but I think this text is better anyway if we just sidestep that issue altogether:

```suggestion
    ::  The ceremony was cancelled by an {{AbortController}}.
        See [[#sctn-abortoperation]] and [[#sctn-sample-aborting]].
```